addons/binding/org.openhab.binding.xmltv/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,24 @@
+Manifest-Version: 1.0
+Automatic-Module-Name: org.openhab.binding.xmltv
+Bundle-ActivationPolicy: lazy
+Bundle-ClassPath: .
+Bundle-ManifestVersion: 2
+Bundle-Name: XmlTV Binding
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-SymbolicName: org.openhab.binding.xmltv;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.4.0.qualifier
+Import-Package: 
+ javax.measure,
```

Comment:
Do we need some imports for `javax.xml...` pacakges here? My first try gives this exception:

```
2018-10-18 11:11:57.179 [ERROR] [g.xmltv.internal.XmlTVHandlerFactory] - Unable to create XmlTVHandler : Provider com.sun.xml.bind.v2.ContextFactory could not be instantiated: javax.xml.bind.JAXBException: "org.openhab.binding.xmltv.internal.jaxb" enthält ObjectFactory.class oder jaxb.index nicht
2018-10-18 11:11:57.185 [ERROR] [core.thing.internal.ThingManagerImpl] - Exception occurred while calling thing handler factory 'org.openhab.binding.xmltv.internal.XmlTVHandlerFactory@5f01ae': XmlTVHandlerFactory could not create a handler for the thing 'xmltv:XmlTVFile:bfd2510a'.
java.lang.IllegalStateException: XmlTVHandlerFactory could not create a handler for the thing 'xmltv:XmlTVFile:bfd2510a'.
...
```